Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Hotfix for BTS report #584669:
- Don't pass through a subtitle as plain text into document body due to a
  missing stylesheet (problem is style independent).
- Display subtitle below title in simple style, just as it is done in db2latex
  style.
--- a/xsl/common/titles.xsl
+++ b/xsl/common/titles.xsl
@@ -81,6 +81,9 @@
   </xsl:choose>
 </xsl:template>
 
+<!-- Don't include subtitles in document body. -->
+<xsl:template match="subtitle"/>
+
 <!-- only occurs in HTML Tables! -->
 <xsl:template match="caption" mode="title.markup">
   <xsl:param name="allow-anchors" select="0"/>
--- a/latex/contrib/example/dbsimple.sty
+++ b/latex/contrib/example/dbsimple.sty
@@ -87,3 +87,12 @@
   \end{minipage}\end{Sbox}\shadowbox{\TheSbox}%
 }
 
+%
+% Subtitle handling
+%
+\let\stdtitle=\title
+\def\title#1{
+  \def\DBKtitle{#1}
+  \stdtitle{\bfseries #1%
+    \ifthenelse{\equal{\DBKsubtitle}{}}{}{\\[12pt]\normalsize \DBKsubtitle}}
+}
